python/doc/usecases/use_case_flood_model.rst

```diff
@@ -124,8 +124,8 @@ by Bayesian methods.
 References
 ----------
 
- * Iooss B, Lemaître P (2015) A review on global sensitivity analysis methods. In: Meloni C., Dellino G. (eds) Uncertainty management in Simulation-Optimization of Complex Systems: Algorithmsand Applications, Springer
- * Baudin M., Dutfoy A., Iooss B., Popelin AL. (2015) OpenTURNS: An Industrial Software for Uncertainty Quantification in Simulation. In: Ghanem R., Higdon D., Owhadi H. (eds) Handbook of Uncertainty Quantification. Springer
+- Iooss B., Lemaître P. (2015) A review on global sensitivity analysis methods. In: Meloni C., Dellino G. (eds) Uncertainty management in Simulation-Optimization of Complex Systems: Algorithmsand Applications, Springer
+- Baudin M., Dutfoy A., Iooss B., Popelin A.-L. (2015) OpenTURNS: An Industrial Software for Uncertainty Quantification in Simulation. In: Ghanem R., Higdon D., Owhadi H. (eds) Handbook of Uncertainty Quantification. Springer
```

Comment:
Shouldn't these references be in the global bibliography anyway?

Reply:
Sufficient and efficient. I have to do that indeed.

Reply:
Ok. Fixed.